apps/agile-compose

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+export AGILE_CLIENT_PORT=2000
+export DBUS_SYSTEM_SOCKET=/var/run/dbus/system_bus_socket
+export DBUS_SESSION_SOCKET=$HOME/agile_bus_socket2
+export ARCH=`arch`
+
+case $ARCH in
+  armv7l )
+    export IMAGE_INFLUXDB=sbiermann/rpi-influxdb
+    ;;
+  x86_64 )
+    export IMAGE_INFLUXDB=influxdb
+    ;;
+  * )
+    export IMAGE_INFLUXDB=influxdb
+    ;;
+esac
+
+docker-compose $@ <<END
```

Comment:
Why pass the full file again? You can just run `docker compose` in the file with the docker-compose.yml